src/components/InfoSection/InfoSection.style.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+const InfoSectionWrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  height: 228px;
```

Comment:
Hard-coding the height here is counter-productive when you consider that there might be another info section element added later on. See my comment in the component below